### PR TITLE
Add integration test for node

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: node/tests/integration_tests.rs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    run_command("cd repo && git init").await;
+    run_command("git commit --allow-empty -m 'initial'").await;
+    simperby_repository::raw::reserved_state::write_reserved_state("./", &rs)
+        .await
+        .unwrap();
+    run_command("git add -A").await;
+    run_command("git commit -m 'genesis'").await;
+}
+
+// TODO: remove this
```
Comment (TomTaehoonKim) [inline node/tests/integration_tests.rs:76]: why is it to be removed?

File: node/tests/integration_tests.rs [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                name: format!("node-{}", i),
+                public_key: c.public_key.clone(),
+                governance_voting_power: 1,
+                consensus_voting_power: 1,
+                governance_delegations: None,
+                consensus_delegations: None,
+            })
+            .chain(std::iter::once(Member {
+                name: "node-relayer".to_owned(),
+                public_key: configs[0].public_key.clone(),
```
Comment (TomTaehoonKim) [inline node/tests/integration_tests.rs:127]: is it possible for a member to perform a role as a node and also a relayer node at the same time?

Comment (TomTaehoonKim) [inline node/tests/integration_tests.rs:127]:
```suggestion
                public_key: configs[4].public_key.clone(),
```

File: node/tests/integration_tests.rs [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        setup_genesis_repo(&config.repository_directory, rs.clone()).await;
+    }
+    push_repo(&configs[0].repository_directory).await;
+
+    let mut proposer_node = initialize(configs[0].clone()).await.unwrap();
+    let mut other_nodes = Vec::new();
+    for config in configs[1..=3].iter() {
+        other_nodes.push(initialize(config.clone()).await.unwrap());
+    }
+    let relay_node = initialize(configs[4].clone()).await.unwrap();
```
Comment (TomTaehoonKim) [inline node/tests/integration_tests.rs:147]: isn't this out of bounds?

File: node/tests/integration_tests.rs [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+// TODO: remove this
+async fn push_repo(path: &str) {
+    run_command(&format!("cd {}", path)).await;
+    run_command("cd repo").await;
+    run_command("git push public_1 -f --all").await;
+}
+
+#[ignore]
```
Comment (TomTaehoonKim) [inline node/tests/integration_tests.rs:83]: why ignored?

File: node/tests/integration_tests.rs (hunk 1 above)
Comment (junha1) [reply node/tests/integration_tests.rs:76]: directly accessing the git repository is not needed after we implement a git server

**Pull request closed without merging**